README.adoc

```diff
@@ -40,20 +40,15 @@ Open Liberty. If you run the application, Maven creates an Open Liberty server a
 the application on it. The application displays a simple web page with a link that, when 
 clicked, calls the servlet to return a simple response of `Hello! How are you today?`.
 
-One benefit of using Maven is that, once you have defined the details of the project and 
-any dependencies it has, Maven automatically handles downloading and installing any and 
-all of the dependencies.
+Maven has some differences and benefits over another build tool called Gradle. 
+Maven defines its builds using `.xml` files in a hierarchical dependency tree. 
+As an older tool Maven also has greater integrations with IDEs. 
+Maven also has an extensive plugin library which offers a variety of ways to quickly customize your build.
+This makes Maven a good choice for smaller, quicker projects. 
```

Comment:
Are you going to be changing the gradle-intro guide as well? Not sure I understand why we are making these changes here? Are we trying to promote Maven over Gradle? I can see adding a couple bits on why we think Maven is a good choice, but I don't understand why we are removing some of the sentences about the benefits of using a build tool period (either Maven or Gradle). The parts about handling dependencies, and automated tests, etc. seem like they should remain.

Reply:
Yes, we will update both Maven and Gradle guides. The reason for changing is because both guides use the same benefit. It does not help users to choice (see https://github.com/OpenLiberty/guide-gradle-intro/issues/11). We just want to update both guides to state their own benefits. There is no intention to compare both or promote either of them.

Reply:
Changes also made to Gradle guide in this PR: https://github.com/OpenLiberty/guide-gradle-intro/pull/73

Added back a paragraph about common build tool benefits such as handling dependencies and automated tests.